### PR TITLE
Update ALL SCRIPTS.vbs

### PR DESCRIPTION
**Pull request by khiestand**
Added Genetic testing invoice button.

### CODE REVIEW

File: ALL SCRIPTS.vbs [hunk 1]
```diff
@@ -237,6 +237,15 @@ cs_scripts_array(script_num).agencies_that_use		= "BELTRAMI, DAKOTA, HENNEPIN, M
 script_num = script_num + 1
 ReDim Preserve cs_scripts_array(script_num)
 SET cs_scripts_array(script_num) = NEW cs_script
+cs_scripts_array(script_num).script_name		= "GENETIC TESTING INVOICE"
+cs_scripts_array(script_num).script_type		= "NOTES"
+cs_scripts_array(script_num).file_name			= "NOTES - GENETIC TESTING INVOICE.vbs"
+cs_scripts_array(script_num).description		= "NEW 06/2016!! Creates CAAD note for recording genetic testing invoices."
+cs_scripts_array(script_num).agencies_that_use		= "BELTRAMI, HENNEPIN, MILLE LACS, OLMSTED, RAMSEY, RENVILLE, SCOTT, STEARNS, WASHINGTON, WRIGHT"
```
Comment (RobertFewins-Kalb) [inline ALL SCRIPTS.vbs]: Is there a reason you left Anoka and Dakota off the list of agencies_that_use?

**Comment by RobertFewins-Kalb** [comment]
If there is a reason why Anoka and Dakota were left off the list, I will accept the pull request. Otherwise, please update. 

NICE JOB, @khiestand!

**Comment by khiestand** [comment]
@RobertFewins-Kalb 
I just copied that from the text above or below, but I would think Anoka and Dakota should also get this one.

**Comment by RobertFewins-Kalb** [comment]
Thanks @khiestand

**Pull request merged**